domain/build.gradle

```diff
@@ -107,3 +124,30 @@ repositories {
 configurations {
   all*.exclude module: 'protobuf-java'
 }
+
+
+tasks.withType(Test) {
+  jacoco.includeNoLocationClasses = true
+}
+task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'createDebugCoverageReport']) {
```

Comment:
Can we somehow share the JaCoCo task across all Gradle files that need it? Also, shouldn't this get included in app, domain, data, utility, & testing module Gradle files? I think you might be missing data.

Reply:
Done